packages/sdk/src/lend/providers/morpho/vaults.ts

```diff
@@ -144,15 +148,51 @@ export function calculateBaseApy(vault: any): number {
  * @returns Promise resolving to detailed vault information
  */
 export async function getVaultInfo(
-  vaultAddress: Address,
+  vaultConfig: Address | LendMarketConfig,
```

Comment:
will change this to two separate param options and require one: `vaultConfig?, address?`